src/NuGet.Clients/NuGet.VisualStudio.Implementation/ExtensionManagerShim.cs

```diff
@@ -4,57 +4,50 @@
 using System;
 using System.Linq;
 using System.Reflection;
+using Microsoft.VisualStudio.ExtensionManager;
 using Microsoft.VisualStudio.Shell;
 using NuGet.VisualStudio.Implementation.Resources;
 
 namespace NuGet.VisualStudio
 {
     internal class ExtensionManagerShim
     {
-        private static Type _iInstalledExtensionType;
-        private static Type _iVsExtensionManagerType;
-        private static PropertyInfo _installPathProperty;
-        private static Type _sVsExtensionManagerType;
-        private static MethodInfo _tryGetInstalledExtensionMethod;
-        private static bool _typesInitialized;
+        private static Type IVsExtensionManagerType;
+        private static Type SVsExtensionManagerType;
+        private static MethodInfo TryGetInstalledExtensionMethod;
+        private static bool TypesInitialized;
```

Comment:
The rest of our code base uses `_` as a prefix for all private fields.

Reply:
Reverted to original names.